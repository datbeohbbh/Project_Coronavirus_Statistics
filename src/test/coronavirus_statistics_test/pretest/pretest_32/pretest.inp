50 100
q 1 2 34
q 1 16 201
? 7
? 4 2
q 1 37 462
? 4 3
q 1 9 385
? 4 2
q 3 2 20
? 4 1
q 3 37 71
q 3 2 10
? 6
q 1 28 154
q 1 15 169
q 3 15 68
q 1 47 19
? 6
? 6
? 4 5
q 2 37 83
? 4 6
q 1 4 359
? 6
q 1 50 358
? 4 1
? 6
? 7
q 1 44 3
q 2 9 194
? 7
q 1 28 370
? 4 2
q 1 5 339
q 3 2 4
? 7
q 2 47 14
q 1 23 400
q 3 50 250
q 1 43 228
q 1 13 305
q 2 5 153
q 3 15 20
? 4 8
q 3 15 80
q 1 9 488
q 3 50 73
q 1 19 262
q 1 29 10
? 6
q 1 8 11
q 2 37 202
? 4 6
q 3 13 82
q 3 16 130
q 1 1 274
q 3 5 134
q 1 2 252
? 6
q 3 44 2
q 2 9 119
q 2 2 35
? 6
? 4 17
q 1 32 121
q 2 28 78
? 4 14
q 2 29 3
q 2 4 99
q 3 1 85
q 3 5 8
q 2 32 96
q 2 16 31
? 4 16
q 1 31 424
? 6
? 4 1
? 6
? 4 2
? 6
? 7
? 6
? 6
? 4 12
? 4 11
? 6
? 4 3
? 4 18
? 6
? 7
? 4 14
? 4 3
? 7
? 4 6
? 6
? 4 4
? 4 9
? 4 15
? 7
? 4 9