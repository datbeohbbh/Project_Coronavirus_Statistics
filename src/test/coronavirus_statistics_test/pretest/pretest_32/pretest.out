Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 2 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 201
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 37 
- Information of the city 37 -
-> Number of current patients : 462
-> Has recovered : 0
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 201
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 14
-> Has recovered : 0
-> Was dead : 20

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 4 5 :
The answer is the city with the index 16 
- Information of the city 16 -
-> Number of current patients : 201
-> Has recovered : 0
-> Was dead : 0

Query ? 4 6 :
The answer is the city with the index 37 
- Information of the city 37 -
-> Number of current patients : 308
-> Has recovered : 83
-> Was dead : 71

Query ? 6 :
The city with the highest recovery is city with the index 37
Quantity: 83 people

Query ? 4 1 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 4
-> Has recovered : 0
-> Was dead : 30

Query ? 6 :
The city with the highest recovery is city with the index 37
Quantity: 83 people

Query ? 7 :
The city with the highest death is city with the index 37
Quantity: 71 people

Query ? 7 :
The city with the highest death is city with the index 37
Quantity: 71 people

Query ? 4 2 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 4
-> Has recovered : 0
-> Was dead : 30

Query ? 7 :
The city with the highest death is city with the index 37
Quantity: 71 people

Query ? 4 8 :
The answer is the city with the index 43 
- Information of the city 43 -
-> Number of current patients : 228
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 9
Quantity: 194 people

Query ? 4 6 :
The answer is the city with the index 50 
- Information of the city 50 -
-> Number of current patients : 35
-> Has recovered : 0
-> Was dead : 323

Query ? 6 :
The city with the highest recovery is city with the index 37
Quantity: 285 people

Query ? 6 :
The city with the highest recovery is city with the index 9
Quantity: 313 people

Query ? 4 17 :
The answer is the city with the index 28 
- Information of the city 28 -
-> Number of current patients : 524
-> Has recovered : 0
-> Was dead : 0

Query ? 4 14 :
The answer is the city with the index 19 
- Information of the city 19 -
-> Number of current patients : 262
-> Has recovered : 0
-> Was dead : 0

Query ? 4 16 :
The answer is the city with the index 19 
- Information of the city 19 -
-> Number of current patients : 262
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 9
Quantity: 313 people

Query ? 4 1 :
The answer is the city with the index 44 
- Information of the city 44 -
-> Number of current patients : 1
-> Has recovered : 0
-> Was dead : 2

Query ? 6 :
The city with the highest recovery is city with the index 9
Quantity: 313 people

Query ? 4 2 :
The answer is the city with the index 15 
- Information of the city 15 -
-> Number of current patients : 1
-> Has recovered : 0
-> Was dead : 168

Query ? 6 :
The city with the highest recovery is city with the index 9
Quantity: 313 people

Query ? 7 :
The city with the highest death is city with the index 50
Quantity: 323 people

Query ? 6 :
The city with the highest recovery is city with the index 9
Quantity: 313 people

Query ? 6 :
The city with the highest recovery is city with the index 9
Quantity: 313 people

Query ? 4 12 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 217
-> Has recovered : 35
-> Was dead : 34

Query ? 4 11 :
The answer is the city with the index 1 
- Information of the city 1 -
-> Number of current patients : 189
-> Has recovered : 0
-> Was dead : 85

Query ? 6 :
The city with the highest recovery is city with the index 9
Quantity: 313 people

Query ? 4 3 :
The answer is the city with the index 47 
- Information of the city 47 -
-> Number of current patients : 5
-> Has recovered : 14
-> Was dead : 0

Query ? 4 18 :
The answer is the city with the index 31 
- Information of the city 31 -
-> Number of current patients : 424
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 9
Quantity: 313 people

Query ? 7 :
The city with the highest death is city with the index 50
Quantity: 323 people

Query ? 4 14 :
The answer is the city with the index 43 
- Information of the city 43 -
-> Number of current patients : 228
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 47 
- Information of the city 47 -
-> Number of current patients : 5
-> Has recovered : 14
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 50
Quantity: 323 people

Query ? 4 6 :
The answer is the city with the index 32 
- Information of the city 32 -
-> Number of current patients : 25
-> Has recovered : 96
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 9
Quantity: 313 people

Query ? 4 4 :
The answer is the city with the index 29 
- Information of the city 29 -
-> Number of current patients : 7
-> Has recovered : 3
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 44
-> Has recovered : 153
-> Was dead : 142

Query ? 4 15 :
The answer is the city with the index 4 
- Information of the city 4 -
-> Number of current patients : 260
-> Has recovered : 99
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 50
Quantity: 323 people

Query ? 4 9 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 44
-> Has recovered : 153
-> Was dead : 142

